submissions/NatalyaBortsova/popup/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/style.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <header class="header">
+    <div class="container">
+      <nav class="header__menu menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a class="menu__link animation" href="#" tabindex="0">
+              Mail
+            </a>
+          </li>
+          <li class="menu__item">
+            <a class="menu__link animation" href="#" tabindex="0">
+              Images
+            </a>
+          </li>
+          <li class="menu__item">
+            <input id="checkbox" class="menu__item-checkbox" type="checkbox">
```

Comment:
[Watch the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide checkbox in the right way